src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -178,7 +178,22 @@ public static class CLIOptions
             ArgumentShortName = "-fte <files-to-exclude>",
             ArgumentDescription = "Set files to exclude for mutation. Example: ['C:\\ExampleProject\\Example.cs','C:\\ExampleProject\\Example2.cs']",
             DefaultValue = null,
-            JsonKey = "files-to-exclude"
+            JsonKey = "files-to-exclude",
+            Deprecated = true,
+            DeprecatedMessage = "Use '--file-pattern' instead."
+        };
+
+        public static readonly CLIOption<string[]> FilePatterns = new CLIOption<string[]>
+        {
+            ArgumentName = "--file-patterns",
```

Comment:
We seem to not have updated the original issue, but we decided on Mutate for the option, as this is what other stryker frameworks use as well:

https://github.com/stryker-mutator/stryker-net/blob/2e479d9d181624c9edccb91431ee9ffad4849be8/src/Stryker.CLI/Stryker.CLI/CLIOptions.cs#L134

https://github.com/stryker-mutator/stryker-net/blob/2e479d9d181624c9edccb91431ee9ffad4849be8/src/Stryker.CLI/Stryker.CLI/CLIOptions.cs#L134